head.html

```diff
@@ -1,4 +1,5 @@
 <meta name="viewport" content="width=device-width, initial-scale=1"/>
+<script src="scripts/purify.min.js" type="text/javascript"></script>
```

Comment:
I think we should dynamically import `purify.min.js` within the `fragment` block to avoid loading it on all pages that don't have any fragments.

Reply:
Good point! Done in [1690e40](https://github.com/adobe/aem-boilerplate/pull/416/commits/1690e4063ccb198de7829d39d4397f77901278d5)